puppet-lint-absolute_classname-check.gemspec

```diff
@@ -1,10 +1,10 @@
 Gem::Specification.new do |spec|
   spec.name        = 'puppet-lint-absolute_classname-check'
-  spec.version     = '0.2.4'
-  spec.homepage    = 'https://github.com/puppet-community/puppet-lint-absolute_classname-check'
+  spec.version     = '0.2.5'
```

Comment:
I've dropped support for ruby 1.8 in the *development* dependencies only, so I think a point release is still fine.